<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QWidget" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя пользователя</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="followersLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 114, 125);</string>
     </property>
     <property name="text">
      <string>Подписчиков: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="followButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="followingLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 114, 125);</string>
     </property>
     <property name="text">
      <string>Подписок: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QLabel" name="pictureLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTabWidget" name="infoTabsWidget">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="overviewTab">
      <attribute name="title">
       <string>Обзор</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="overviewScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="overviewScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tracksTab">
      <attribute name="title">
       <string>Любимые треки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tracksTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="playlistsTab">
      <attribute name="title">
       <string>Плейлисты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="playlistsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="playlistsScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="albumsTab">
      <attribute name="title">
       <string>Альбомы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="albumsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="albumsScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="artistsTab">
      <attribute name="title">
       <string>Исполнители</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="artistsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="artitsScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="podcastsTab">
      <attribute name="title">
       <string>Подкасты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QScrollArea" name="podcastsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="podcastsScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="radiosTab">
      <attribute name="title">
       <string>Миксы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="radiosScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="radiosScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="followingTab">
      <attribute name="title">
       <string>Подписки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QScrollArea" name="followingScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="followingScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="followersTab">
      <attribute name="title">
       <string>Подписчики</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QScrollArea" name="followersScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="followersScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>146</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
